<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QMainWindow" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CompSecMonSys - Client</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/icon.png</normaloff>img/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>191</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>üîç</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser_errors">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_auto_search">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>–ê–≤—Ç–æ–ø–æ–∏—Å–∫ —Å–µ—Ä–≤–µ—Ä–∞</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_count_q">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>üì°–ö–æ–ª–∏—á–µ—Å—Ç–≤–æ –æ—Ç–ø—Ä–∞–≤–µ–Ω–Ω—ã—Ö –∑–∞–ø—Ä–æ—Å–æ–≤: 0</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_time">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_localhost">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>localhost</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>üìù</string>
     </attribute>
     <widget class="QCheckBox" name="checkBox_net_con">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>–°–µ—Ç–µ–≤—ã–µ –∏–Ω—Ç–µ—Ä—Ñ–µ–π—Å—ã</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_port">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>–ü–æ—Ä—Ç—ã</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_brandmayer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>–ë—Ä—ç–Ω–¥–º–∞—É—ç—Ä</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_antivirus">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Antivirus</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_active_con">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>–ê–∫—Ç–∏–≤–Ω—ã–µ —Å–æ–µ–¥–∏–Ω–µ–Ω–∏—è</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_create_info">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>–°–æ–∑–¥–∞—Ç—å</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_sys_info">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>–°–∏—Å—Ç–µ–º–Ω–∞—è –∏–Ω—Ñ–æ—Ä–º–∞—Ü–∏—è</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_errors_info">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>171</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_process">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>–ü—Ä–æ—Ü–µ—Å—Å—ã</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>üì§</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser_errors_send">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>121</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolBox" name="toolBox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>191</width>
        <height>141</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>98</width>
         <height>28</height>
        </rect>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <attribute name="label">
        <string>–°–µ—Ä–≤–µ—Ä</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_send_file">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>–ü–µ—Ä–µ–¥ –æ—Ç–ø—Ä–∞–≤–∫–æ–π —Ñ–∞–π–ª–∞, —É–±–µ–¥–∏—Ç–µ—Å—å, —á—Ç–æ —Å–µ—Ä–≤–µ—Ä –≥–æ—Ç–æ–≤ –ø—Ä–∏–Ω—è—Ç—å —Ñ–∞–π–ª</string>
        </property>
        <property name="text">
         <string>–û—Ç–ø—Ä–∞–≤–∏—Ç—å</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>191</width>
         <height>87</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Telegram</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_save_tele_id">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>0</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>–°–æ—Ö—Ä–∞–Ω–∏—Ç—å</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_send_tele">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>171</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>–û—Ç–ø—Ä–∞–≤–∏—Ç—å</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_tele_id">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLineEdit" name="lineEdit_send_file">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>üìÑ–§–∞–π–ª, –∫–æ—Ç–æ—Ä—ã–π –±—É–¥–µ—Ç –æ—Ç–ø—Ä–∞–≤–ª–µ–Ω</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>1000</number>
      </property>
      <property name="placeholderText">
       <string>C:\</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_set_file">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>–í—ã–±—Ä–∞—Ç—å</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>ü§ñ</string>
     </attribute>
     <widget class="QCheckBox" name="checkBox_active_con_auto">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>–ê–∫—Ç–∏–≤–Ω—ã–µ —Å–æ–µ–¥–∏–Ω–µ–Ω–∏—è</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_autostart_auto">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_net_con_auto">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>–°–µ—Ç–µ–≤—ã–µ –∏–Ω—Ç–µ—Ä—Ñ–µ–π—Å—ã</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_process_auto">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>–ü—Ä–æ—Ü–µ—Å—Å—ã</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_port_auto">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>–ü–æ—Ä—Ç—ã</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_save_auto">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>–ü—Ä–∏–º–µ–Ω–∏—Ç—å</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_antivirus_auto">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Antivirus</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>-10</y>
        <width>20</width>
        <height>171</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_brandmayer_auto">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>–ë—Ä—ç–Ω–¥–º–∞—É—ç—Ä</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_sys_info_auto">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>–°–∏—Å—Ç–µ–º–Ω–∞—è –∏–Ω—Ñ–æ—Ä–º–∞—Ü–∏—è</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_telegram_auto">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Telegram ‚úàÔ∏è</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
